src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenSilentParameterBuilder.cs

```diff
@@ -195,16 +196,20 @@ public AcquireTokenSilentParameterBuilder WithProofOfPossession(string nonce, Ht
             popConfig.HttpMethod = httpMethod ?? throw new ArgumentNullException(nameof(httpMethod));
             popConfig.Nonce = nonce;
 
+            IAuthenticationScheme authenticationScheme;
+
             //POP Auth scheme should not wrap and sign token when broker is enabled for public clients
             if (ServiceBundle.Config.IsBrokerEnabled)
             {
                 popConfig.SignHttpRequest = false;
+                authenticationScheme = new PopBrokerAuthenticationScheme();
+            }
+            else
+            {
+                authenticationScheme = new PoPAuthenticationScheme(popConfig, ServiceBundle);
```

Comment:
can this be a consistent name? PoP / Pop. Pop is better?

Reply:
`Pop`. 👍